lib/src/responder/responder.dart

```diff
@@ -436,7 +436,6 @@ class Responder extends ConnectionHandler {
         } else {
           closeResponse(m['rid'], error: DSError.PERMISSION_DENIED);
         }
-        closeResponse(m['rid']);
```

Comment:
Now the other case of the IF branch never calls close response?

Prior to this, after the node.setValue happened on line 435, then closeResponse would be called (without any error). Now it never sends the close if the value is set.

Reply:
https://github.com/IOT-DSA/sdk-dslink-dart/blob/master/lib/src/responder/node_provider.dart#L220

we pass response reference to the setValue API because we need to let the callback decide when and how to close the response, like with a delay, or return error.